lib/android/app/src/main/java/com/reactnativenavigation/options/TitleOptions.java

```diff
@@ -47,7 +47,14 @@ public static TitleOptions parse(Context context, TypefaceLoader typefaceManager
     public Number topMargin = new NullNumber();
 
     void mergeWith(final TitleOptions other) {
-        if (other.text.hasValue()) text = other.text;
+        if (other.text.hasValue()) {
+            text = other.text;
+            this.component = new ComponentOptions();
```

Comment:
why do we need to create `ComponentOptions` instance here? it doesn't feel right

Reply:
we need to clear them, RNN throws error invalidate about having component and text set simultaneously, that means one of component and test must be unset (null, clean) at the same time the other has value.